examples/tutorials/api/irfs.py

```diff
@@ -0,0 +1,495 @@
+"""
+Using Gammapy IRFs
+==================
+
+`gammapy.irf` contains classes for handling Instrument Response
+Functions typically stored as multi-dimensional tables. For a list of
+IRF classes internally supported, see
+https://gamma-astro-data-formats.readthedocs.io/en/v0.3/irfs/full_enclosure/index.html
+
+This tutorial is intended for advanced users typically creating IRFs.
+
+"""
+
+import numpy as np
+import scipy.special
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import quantity_support
+import matplotlib.pyplot as plt
+from gammapy.irf import (
+    IRF,
+    Background3D,
+    EffectiveAreaTable2D,
+    EnergyDependentMultiGaussPSF,
+    EnergyDispersion2D,
+)
+from gammapy.irf.io import COMMON_IRF_HEADERS, IRF_DL3_HDU_SPECIFICATION
+from gammapy.makers.utils import make_edisp_kernel_map, make_map_exposure_true_energy
+from gammapy.maps import MapAxes, MapAxis, WcsGeom
+
+######################################################################
+# Inbuilt Gammapy IRFs
+# --------------------
+#
+
+irf_filename = (
+    "$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"
+)
+
+aeff = EffectiveAreaTable2D.read(irf_filename, hdu="EFFECTIVE AREA")
+print(aeff)
+
+
+######################################################################
+# We can see that the Effective Area Table is defined in terms of
+# `energy_true` and `offset` from the camera center
+#
+
+# To see the IRF axes binning, eg, offset
+print(aeff.axes["offset"])
+
+# To get the IRF data
+print(aeff.data)
+
+# the aeff is evaluated at a given energy and offset
+print(aeff.evaluate(energy_true=[1, 10] * u.TeV, offset=[0.2, 2.5] * u.deg))
+
+
+# The peek method gives a quick look into the IRF
+aeff.peek()
+
+
+######################################################################
+# Similarly, we can access other IRFs as well
+#
+
+psf = EnergyDependentMultiGaussPSF.read(irf_filename, hdu="Point Spread Function")
+bkg = Background3D.read(irf_filename, hdu="BACKGROUND")
+edisp = EnergyDispersion2D.read(irf_filename, hdu="ENERGY DISPERSION")
+
+print(bkg)
+
+
+######################################################################
+# Note that the background is given in FoV coordiantes with `fov_lon`
+# and `fov_lat` axis, and not in `offset` from the camera center. We
+# can also check the Field of view alignment. Currently, two possible
+# alignments are supported: alignment with the horizontal coordinate
+# system (ALTAZ) and alignment with the equatorial coordinate system
+# (RADEC).
+#
+
+bkg.fov_alignment
+
+
+######################################################################
+# To evaluate the IRFs, pass the values for each axis. To know the default
+# interpolation scheme for the data
+#
+
+bkg.interp_kwargs
+
+# Evaluate background
+energy = [1, 10, 100, 1000] * u.TeV
+fov_lon = [
+    1,
+    2,
+] * u.deg
+fov_lat = [1, 2] * u.deg
+ev = bkg.evaluate(
```

Comment:
Maybe add a comment explaining that evaluation supports broadcasting numpy-style

As the command might look cryptic